Remotion/Web/Development.WebTesting/Configuration/WebTestSettings.cs

```diff
@@ -57,11 +63,35 @@ public static IWebTestSettings Current
       }
     }
```

Comment:
Something I missed in the previous PR: for cases when setting the current value is a very conscious act, I prefer having a SetCurrent method instead